source/Coop/Mod/Serializers/SerializableFactory.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Sync.Store;
+using TaleWorlds.CampaignSystem;
+
+namespace Coop.Mod.Serializers
+{
+    /// <summary>
+    ///     Factory to create the serialization wrappers for non-serializable game objects.
+    /// </summary>
+    public class SerializableFactory : ISerializableFactory
```

Comment:
Doesn't contain all serializer members. Should I handle that in #45?

Reply:
I'd appreciate if you could do that.